main.css

```diff
@@ -0,0 +1,50 @@
+@import url('https://fonts.googleapis.com/css?family=Merriweather:900|Roboto');
+
+* {
+    width: 625px;
+    color: white;
```

Comment:
Adding the line `margin: 0 auto;` here will center the entire structure horizontally.